### DISCUSSION TITLE
[논의] StoreRegistrationService 파라미터

### DISCUSSION THREAD

**Discussion by kimhyun5u** [woowa-techcamp-2024/Team1-3K1K#42] [General]
### 배경

https://github.com/woowa-techcamp-2024/Team1-3K1K/blob/882229067bd2f01674d9c0c87a2ae569136b7388/src/main/java/camp/woowak/lab/store/service/StoreRegistrationService.java#L28

에 대한 결정

### 세부 주제

```bash
1. 레이어 간 데이터 객체 추상화의 필요성
```

**Comment by Dr-KoKo** [comment]
Session에 `LoginMemer`를 저장 vs `Vendor`를 저장 하는 문제인 것 같습니다.
Session에 저장되어 있는 Vendor(Entity)는 영속성 컨텍스트에서 관리되지 않는 상태이기 때문에 실수(ex 영속성 컨텍스트로 관리되고 있지 않은 Vendor를 수정하는 Service)를 방지하기 위해서 저는 개인적으로 `LoginMember`와 같은 별도의 모델을 사용하는 편입니다.
